server/etcdmain/etcd.go

```diff
@@ -465,6 +465,7 @@ func checkSupportArch() {
 	// to add a new platform, check https://github.com/etcd-io/website/blob/main/content/en/docs/next/op-guide/supported-platform.md
```

Comment:
Please update the link to: https://etcd.io/docs/v3.5/op-guide/supported-platform. 

It requires adding CI test for build to make it tier-3 'unstable' platform. 

Personally I think we should keep warning for  any setup that is not "Tier 1 or 2".